RetailCoder.VBE/Inspections/Resources/InspectionsUI.resx

```diff
@@ -647,7 +647,7 @@ If the parameter can be null, ignore this inspection result; passing a null valu
   <data name="ApplicationWorksheetFunctionQuickFix" xml:space="preserve">
     <value>Use Application.WorksheetFunction explicitly.</value>
   </data>
-  <data name="AssignedByValParameterQuickFix" xml:space="preserve">
+  <data name="AssignedByValParameterMakeLocalCopyQuickFix" xml:space="preserve">
```

Comment:
There were not pre-existing entries in fr.resx and de.resx to replace for 'AssignedByValParameterQuickFix'.  So, I'm planning to add the entry with the english description unless someone can provide translations...Or, if it is better to leave them missing than entered in english.  Thanks.